deploy/standard/operator.yaml

```diff
@@ -31,7 +31,7 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
-        image: rthallisey/hyperconverged-cluster-operator:latest
+        image: docker.io/rthallisey/hyperconverged-cluster-operator:latest
```

Comment:
You can't just change this here. You'll either need to make changes [here](https://github.com/kubevirt/hyperconverged-cluster-operator/blob/master/pkg/components/components.go#L16) **or, probably easier,** update the [manifest-templator](https://github.com/kubevirt/hyperconverged-cluster-operator/blob/master/tools/manifest-templator/manifest-templator.go#L149). After you update the templator (or components.go) you'll need to re-run `./hack/build-manifests.sh`